README.md

```diff
@@ -58,3 +58,23 @@ Commands:
   version                   prints the om release version
 
 ```
+## Authentication
+OM will by preference use Client ID and Client Secret if provided. To create a Client ID and Client Secret           
+                                                             
+1. `uaac target https://YOUR_OPSMANAGER/uaa`
+1. `uaac token sso get` if using SAML or `uaac token owner get` if using internal auth
+1. Generate a client ID and secret
+```
+uaac client add -i
+Client ID:  NEW_CLIENT_NAME
+New client secret:  DESIRED_PASSWORD
+Verify new client secret:  DESIRED_PASSWORD
+scope (list):  opsman.admin
+authorized grant types (list):  client_credentials
+authorities (list):  opsman.admin
+access token validity (seconds):  43200
+refresh token validity (seconds):  43200
+redirect uri (list):
+autoapprove (list):
+signup redirect url (url):```
```

Comment:
Hm, I tested it in GitHub wiki and it rendered right, but I put it on its own line neverthetheless.